packages/next/client/index.js

```diff
@@ -257,6 +263,7 @@ async function doRender ({ App, Component, props, err }) {
   ) {
     const { pathname, query, asPath } = router
     props = await loadGetInitialProps(App, {
+      AppContainer,
```

Comment:
This would have to be the same as above, right? `AppTree` -- maybe you should pull this out to a `const` var.

Reply:
updated